src/AuthContext.tsx

```diff
@@ -118,11 +118,15 @@ export const AuthProvider = ({ authConfig, children }: IAuthProvider) => {
     const refreshTokenExpiresIn = config.refreshTokenExpiresIn ?? getRefreshExpiresIn(tokenExpiresIn, response)
     setRefreshTokenExpire(epochAtSecondsFromNow(refreshTokenExpiresIn))
     setIdToken(response.id_token)
+    try {
+      if (response.id_token) setIdTokenData(decodeJWT(response.id_token))
+    } catch (e) {
+      setError(`Failed to decode idToken: ${(e as Error).message}`)
```

Comment:
Note that this is also a breaking change, because it might now get an error trying to decode it, and setting this error in the state will show an error in all applications that prioritize authErrors.

Reply:
Agreed. Exposed "error" value should only be used for fatal errors.
Did a `console.warn` here instead